features/form.feature

```diff
@@ -1,61 +1,5 @@
 Feature: Filling in the form
   In order to get help during the Covid-19 pandemic
-  As an extremely vulnerable person
-  I want to be able to tell my government that I need help
-  So that they can support me
-
-  Scenario: Completes the form
-    When I visit the "/live-in-england" path
-    Then I can see "Do you live in England?" in the heading
-    When I answer "Yes"
-    Then I can see "Have you recently had a letter from the NHS" in the heading
-    When I answer "Yes"
-    Then I can see "Do you have a medical condition" in the heading
-    When I answer "Yes"
-    Then I can see "What is your name?" in the heading
-    When I fill in my name
-    Then I can see "What is your date of birth?" in the heading
-    When I fill in my date of birth
-    Then I can see "What is the address" in the heading
-    When I fill in my address
-    Then I can see "What are your contact details?" in the heading
-    When I fill in my contact details
-    Then I can see "Do you know your NHS number?" in the heading
-    When I answer "Yes"
-    Then I can see "What is your NHS number?" in the heading
-    When I fill in my nhs number
-    Then I can see "Do you have a way of getting essential supplies" in the heading
-    When I answer "No"
-    Then I can see "Are your basic care needs being met" in the heading
-    When I answer "No"
-    Then I can see "Do you have any special dietary requirements?" in the heading
-    When I answer "No"
-    Then I can see "Is there someone in the house who’s able to carry" in the heading
-    When I answer "No"
-    Then I can see "Are you ready to send your application?" in the heading
-    When I click the "Accept and send" button
-    Then I can see "Registration complete"
-
-  Scenario: Visits an intermediate question without having previously visited the form
-    When I visit the "/essential-supplies" path
-    Then I will be redirected to the "/live-in-england" path
-
-  Scenario: Visits the privacy policy
```

Comment:
I think we can keep this test.

Reply:
Oh yeah - good point!

Reply:
Ah, we can't - the first thing it does is got to /live-in-england, and we don't have an equivalent of that yet because I missed all the questions.

Could make it simpler and just go to `/privacy`, but that involves changing the test so let's do it in a separate PR I think.

Reply:
Were you going to add this back in?

Reply:
I can't without rewriting the test - it tries to go to /live-in-england